client/Stars.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Checked = styled.div`
+  color: #da3743;
+  width: 16px;
+  height: 16px;
+  margin-right: 0.25rem;
+`
+
+const HalfCheck = styled.div`
+  color: #da3743;
+  width: 16px;
+  height: 16px;
+  margin-right: 0.25rem;
+`
+
+const UnChecked = styled.div`
+  color: #6f737b;
+  width: 16px;
+  height: 16px;
+  margin-right: 0.25rem;
+`
+
+class Stars extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return(
+      // TODO: CREATE STARS WITHOUT EXTERNAL LIB, ACCOUNT FOR RATINGS LOWER THAN 3
```

Comment:
This can be cleaned up, along with the commented-out block down below.

Reply:
Good catch!